### DISCUSSION TITLE
Error and status pages

### DISCUSSION THREAD

**Discussion by jrbarnes9** [ONSdigital/design-system#237] [Feedback]
Use this place to discuss the [error and status pages pattern](https://ons-design-system.netlify.com/patterns/error-status-pages) in the [ONS Design System](https://ons-design-system.netlify.com)

**Comment by OscarJackson** [comment]
Need to update this for use cases where the page is intentionally unavailable. Similar to in the [GOV.UK service unavailable pattern](https://design-system.service.gov.uk/patterns/service-unavailable-pages/).

I will look at the documentation when I review the pattern.

**Comment by pau1ross** [comment]
do we have Welsh language versions of the Error & Status Pages?

**Comment by Al-Davies** [comment]
#Error Messages UR
Tested as part of eQ user research session on 30/10/23 - users were shown generic error message for business and social surveys to see if they understood the message, and what actions they would take as a result of the message. Error image attached

* If we tell users that the error is likely due to an outdated browser, they will spend time trying to update this (this is a potential waste of time for users)
* When seeing any error message when part way through a survey, a users main concern is will be "what happened to all of the answers I've already put into this survey" - we need to be clear that any answers already completed will be saved
* If we aren't clear about progress being saved, this will likely result in calls to the survey teams to query what to do next, or users becoming frustrated thinking they have to start from scratch
* If error messages are vague, it doesn't help users in knowing what to do next - they feel there isn't enough information to make it worth while speaking to their own IT team
![error message](https://github.com/ONSdigital/design-system/assets/60258560/4c1abe68-c850-42e9-ad16-4096423ea6b8)
